package-templates/basic-01/{{project-name | upper_camel_case}}/Cargo.toml.liquid

```diff
@@ -8,6 +8,17 @@ rust-version = "1.64"
 edition = "2021"
 publish = false
 
+[workspace.dependencies]
+# Pin cxx-build to avoid 1.0.187 which requires Rust 1.87
+cxx-build = ">=1, <1.0.187"
```

Comment:
If this cxx-build is needed, shouldn't it be in the general [packages/X/Cargo.toml](https://github.com/gofractally/psibase/pull/1494/files#diff-c1b6c1bacada5875c47421bb02f2636c504b4011ec5eb4fbfe339268b1bb0e39)?